src/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+    <title>To-do List</title>
+</head>
+<body>
+    <section class="container">
+        <h1 class="center-title">To-Do List</h1>
+        <div class="list-container list-group">
+            <div class="list-head list-group-item p-4 bg-light clearfix">
+                <span class="list-title float-left">today's To-Do</span>
+                <span class="text-right float-right"><i class="fas fa-sync-alt"></i></span>
+            </div>
+            <div class="list-group-item p-3 bg-white">
+                <div class="form-group mb-0 col-lg-12 p-1">
+                    <input type="text" class="form-control border-0 todoInput" id="todoInput" placeholder="Add to your list...">
+                </div>
+            </div>
+                <ul class="w-100 list-group bg-white" id="listWrapper">
+                    <li class="list-group-item pl-5 pt-4 pb-4 clearfix">                        
+                        <input type="checkbox" class="form-check-input float-left">
+                        <span>Do the dishes / Sample element</span>
+                        <span class="float-right"><i class="fas fa-ellipsis-v"></i></span>
+                    </li>
+                    <li class="list-group-item pl-5 pt-4 pb-4 clearfix">                        
+                        <input type="checkbox" class="form-check-input float-left">
+                        <span>Do the laundry / Sample element</span>
+                        <span class="float-right"><i class="fas fa-ellipsis-v"></i></span>
+                    </li>
+                    <li class="list-group-item pl-5 pt-4 pb-4 clearfix">                        
+                        <input type="checkbox" class="form-check-input float-left">
+                        <span>Walk the dog / Sample element</span>
+                        <span class="float-right"><i class="fas fa-ellipsis-v"></i></span>
+                    </li>
```

Comment:
- [ ] I think that we don't actually have to hard code the list items ourselves since all this logic should be in the javascript file, we will need to generate the list of todos according to what we have in the array of objects in the javascript file. See screenshot below 👇 

![image](https://user-images.githubusercontent.com/55465133/134431360-045fd206-d164-44da-a01d-637e1ff7c4c3.png)

![image](https://user-images.githubusercontent.com/55465133/134430402-1532759f-ab18-4c43-a048-73f596bc8cd2.png)

Kindly let's fix this and have this logic in javascript instead 👍